sub-projects.md

```diff
@@ -0,0 +1,13 @@
+# ORAS Sub-Projects
+
+The ORAS Project represents the following sub-projects:
+
+| Sub-Project | Repositories | Sub-Project Maintainers |
+| - | - | - |
+| Org Governance | [governance](https://github.com/oras-project/community) | [CODEOWNERS](https://github.com/oras-project/community/blob/main/CODEOWNERS) | 
+| [oras.land](https://oras.land) website content | [oras-www](https://github.com/oras-project/oras-www) | [CODEOWNERS](https://github.com/oras-project/oras-www/blob/main/CODEOWNERS) |
+| oras go client | [oras](https://github.com/oras-project/oras)<br>[oras-go](https://github.com/oras-project/oras-go)<br>[setup-oras](https://github.com/oras-project/setup-oras)<br>[oras-credentials-go](https://github.com/oras-project/oras-credentials-go) | [CODEOWNERS](https://github.com/oras-project/oras/blob/main/CODEOWNERS) <br>[CODEOWNERS](https://github.com/oras-project/oras-go/blob/main/CODEOWNERS)<br>[CODEOWNERS](https://github.com/oras-project/setup-oras/blob/main/CODEOWNERS)<br>see issue: [Update CODEOWNERS #6](https://github.com/oras-project/oras-credentials-go/issues/6) |
```

Comment:
```suggestion
| oras go client | [oras](https://github.com/oras-project/oras)<br>[oras-go](https://github.com/oras-project/oras-go)<br>[setup-oras](https://github.com/oras-project/setup-oras)<br>[oras-credentials-go](https://github.com/oras-project/oras-credentials-go) | [CODEOWNERS](https://github.com/oras-project/oras/blob/main/CODEOWNERS) <br>[CODEOWNERS](https://github.com/oras-project/oras-go/blob/main/CODEOWNERS)<br>[CODEOWNERS](https://github.com/oras-project/setup-oras/blob/main/CODEOWNERS)<br>see issue: [Update CODEOWNERS](https://github.com/oras-project/oras-credentials-go/pull/15) |
```

Reply:
Thanks @TerryHowe 
Since the issues and PRs are now closed/merged, I'll remove this comment.